include/ftl/task_scheduler.h

```diff
@@ -312,6 +315,10 @@ class TaskScheduler {
 		m_emptyQueueBehavior.store(behavior, std::memory_order_relaxed);
 	}
 
+	inline ISchedulerUserCallbacks *UserCallbacks() {
```

Comment:
Question: Should this return a const, to enforce that the user can't modify the callbacks this way